frontend/public/profile.html

```diff
@@ -3,9 +3,28 @@
 <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Profile</title>
+	<link rel="stylesheet" href="/styles.css">
+    <title>AChat Profile</title>
 </head>
+
 <body>
-    <h1>Profile page</h1>
+    <div class="welcomeMessage" id="welcomeMessage">Profile Page</div>
+	<div><input type="button" class="chatroomButton" id="button" name="button" value="Join Chatroom" onclick="window.location.href='/chat.html'"></div>
+	<div><input type="button" class="helpButton" id="button" name="button" value="Help" onclick="window.location.href='/help.html'"></div>
 </body>
-</html>
\ No newline at end of file
+
+<script>
+	const pathSegments = window.location.pathname.split('/');
+	const username = pathSegments[pathSegments.length - 1];
+
+	console.log("Extracted username:", username);
```

Comment:
I'm unsure of the backend implementation of the `/profile/username` route at the moment, however I believe we will be using a user_id instead. We will figure out soon.